modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -37,29 +37,49 @@ helm get values <chart-name> --namespace <namespace> --all
 
 == View Redpanda logs
 
-Logs for each Redpanda broker running inside a Pod are sent to STDOUT. Use `kubectl logs` to get logs from STDOUT.
+Logs are crucial for monitoring and troubleshooting your Redpanda clusters. Redpanda brokers output logs to STDOUT, making them accessible via `kubectl`.
 
-To view logs for a particular Pod:
+To access logs for a specific Pod:
 
-```bash
-kubectl logs <pod-name> --namespace <namespace>
-```
+. List all Pods to find the names of those that are running Redpanda brokers:
++
+[source,bash]
+----
+kubectl get pods --namespace <namespace>
+----
 
-To view logs for all Pods in the StatefulSet:
+. View logs for a particular Pod by replacing `<pod-name>` with the name of your Pod:
++
+[source,bash]
+----
+kubectl logs <pod-name> --namespace <namespace>
+----
++
+[TIP]
+====
+For a comprehensive overview, you can view aggregated logs from all Pods in the StatefulSet:
 
-```bash
+[source,bash]
+----
 kubectl logs --namespace <namespace> -l app.kubernetes.io/component=redpanda-statefulset
-```
+----
+====
+
+=== Change the log level
 
-To change the log level of the Redpanda brokers to `debug`:
+Changing the log level to `debug` can provide more detailed logs for diagnostics. This logging level increases the volume of generated logs.
+
+Valid values are `trace`, `debug`, `info`, `warn`, `error`.
```

Comment:
```suggestion
Valid values are `trace`, `debug`, `info`, `warn`, `error`.

NOTE: Instead of setting a global log level, you can also <<Set log levels for Redpanda subsystems>>.
```